src/Sarif/CodeGenHints.json

```diff
@@ -50,6 +50,23 @@
       "kind": "DictionaryHint"
     }
   ],
+  "Rule.DefaultLevel": [
```

Comment:
Of course, what we'd really like to do is to create a single enum, ResultLevel, but the serializer doesn't support this.